.github/workflows/label_merge_conflicts.yml

```
@@ -0,0 +1,11 @@
+name: 'Check for merge conflicts'
+on:
+  push
+jobs:
+  triage:
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-18.04
```
We should lock the version, just in case.